pkg/resources/poddistribution/poddistribution.go

```diff
@@ -245,7 +245,7 @@ func forceRebalance(ctx context.Context, client k8sclient.Client, knn *KindNameS
 
 	deletePod(ctx, client, pods[0], knn.Namespace)
 	// The wait prevents version clash errors when updating the controller
-	err := wait.Poll(time.Second*5, time.Second*5, func() (done bool, err error) {
+	err := wait.PollUntilContextTimeout(context.TODO(), time.Second*5, time.Second*5, false, func(ctx2 context.Context) (done bool, err error) {
```

Comment:
ctx2?

Reply:
IDE sets it as ctx2 automatically if ctx is present in method signature. Seems it's ok. We can see both accurances :
func(ctx2 context.Context) and func(ctx context.Context) ..  The second one - in methods were ctx is not present  in signature.